9/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM ubuntu:16.04
 
-ADD config/redis.config /etc/redis/redis.config
-ADD start /start
+COPY config/redis.config /etc/redis/redis.config
```

Comment:
How often do you plan to change those files? If you plan to change them regularly, this should be an image based on this image rather than included in the base.